docs/webpack-chain.md

```diff
@@ -523,37 +528,37 @@ _NOTE: Do not use `new` to create the plugin, as this will be done for you._
 ```js
 neutrino.config
   .plugin(name)
-  .use(WebpackPlugin, args)
+    .use(WebpackPlugin, args)
```

Comment:
Some of these indentation fixes are also applicable to the upstream webpack-chain docs and could be upstreamed at some point :-) (I'm using DiffMerge to do partial line comparisons between the two)

Reply:
Grr, I thought I caught all those there. 😭